files/fr/web/api/clipboard_api/index.html

```diff
@@ -0,0 +1,61 @@
+---
+title: API Clipboard
+slug: Web/API/Clipboard_API
+translation_of: 'Web/API/Clipboard_API'
+---
+<div>{{DefaultAPISidebar("Clipboard API")}}</div>
+
+<p>l'<strong>API Clipboard</strong> (en français : API Presse-papiers) fournit la possibilité de répondre aux commandes du presse-papiers (couper, copier et coller) ainsi que de lire et écrire sur le presse-papiers système de façon asynchrone. L'accès aux contenus du presse-papiers est protégé par l'<a href="/fr/docs/Web/API/Permissions_API">API Permissions</a> : la permission <code>clipboard-write</code> est donnée automatiquement aux pages lorsqu'elles sont dans l'onglet actif. La permission <code>clipboard-read</code> doit quant à elle être demandée, ce que vous pouvez faire en tentant de lire les données du presse-papiers.</p>
+
+<div class="notecard note">
+<p><strong>Note :</strong> Cette API <em>n'est pas disponible</em> dans les <a href="/fr/docs/Web/API/Web_Workers_API">Web Workers</a> (elle n'est pas exposée via {{domxref("WorkerNavigator")}}).</p>
+</div>
+
+<p>Cette API est conçue pour remplacer l'accès au presse-papiers précédemment proposé via {{domxref("document.execCommand()")}}.</p>
+
+<h2 id="accessing_the_clipboard">Accéder au presse-papier</h2>
+
+<p>Au lieu de créer un objet <code>Clipboard</code> lors de l'instantiation, vous pouvez accéder au presse-papier du système avec la globale {{domxref("Navigator.clipboard")}} :</p>
```

Comment:
minor typo (detection sponsored by Grammalecte): instantiation > instanciation

Reply:
thanks, typo fixed in https://github.com/mdn/translated-content/pull/1770/commits/006b650c77390ce8185d51fb09d2f3c0345f4bd2